os/hal/ports/NRF5/NRF51822/hal_pwm_lld.c

```diff
@@ -63,19 +70,32 @@ PWMDriver PWMD3;
 /* Driver local variables and types.                                         */
 /*===========================================================================*/
 
+static const uint8_t pwm_margin_by_prescaler[] = {
+    (PWM_GPIOTE_DECISION_TIME +   0) >> 0,
+    (PWM_GPIOTE_DECISION_TIME +   1) >> 1,
+    (PWM_GPIOTE_DECISION_TIME +   3) >> 2,
+    (PWM_GPIOTE_DECISION_TIME +   7) >> 3,
+    (PWM_GPIOTE_DECISION_TIME +  15) >> 4,
+    (PWM_GPIOTE_DECISION_TIME +  31) >> 5,
+    (PWM_GPIOTE_DECISION_TIME +  63) >> 6,
+    (PWM_GPIOTE_DECISION_TIME + 127) >> 7,
+    (PWM_GPIOTE_DECISION_TIME + 255) >> 8,
+    (PWM_GPIOTE_DECISION_TIME + 511) >> 9

```

Comment:
can leave a ',' at the line ending so future adds don't need to change this line.